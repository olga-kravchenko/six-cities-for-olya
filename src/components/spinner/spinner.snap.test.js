import React from 'react';
import {render} from '@testing-library/react';
import Spinner from "./spinner";

test(`Should Spinner render correctly`, () => {
  const {container} = render(<Spinner />);
  expect(container).toMatchSnapshot();
});
