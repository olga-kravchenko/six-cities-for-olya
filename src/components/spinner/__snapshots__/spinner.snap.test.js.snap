// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should LoadingScreen render correctly 1`] = `
<div>
  <div
    class="container-loader"
  >
    <div
      class="item-loader"
    >
      <div
        class="loader01"
      />
    </div>
  </div>
</div>
`;

exports[`Should Spinner render correctly 1`] = `
<div>
  <div
    class="container-loader"
  >
    <div
      class="item-loader"
    >
      <div
        class="loader01"
      />
    </div>
  </div>
</div>
`;
